pkg/controller/download_controller.go

```diff
@@ -100,6 +100,17 @@ func (c *downloadController) enqueueDownloadItem(obj interface{}) {
 	switch req.Status.Phase {
 	case "", pluginv1api.DownloadPhaseNew, pluginv1api.DownloadPhaseInProgress, pluginv1api.DownLoadPhaseRetry:
 		// Process New InProgress and Retry Downloads
+	case pluginv1api.DownloadPhaseCompleted:
+		// If Download CR status reaches terminal state more than 1 hour, Download CR should be deleted
+		now := c.clock.Now()
+		if req.Status.CompletionTimestamp.Add(time.Hour).Unix() < now.Unix() {
+			log.Infof("Download CR %s reaches phase %v more than 1 hour, deleting this CR.", req.Name, req.Status.Phase)
```

Comment:
Let's change this to
"Download CR %s has been in phase %s more than 1 hour, deleting this CR."